CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 # Unreleased
 
+# 9.14.2 (Patch release)
+
 ## Fixes
 
-- [Pull request #1050: Don’t swallow errors from session-data-defaults.js](https://github.com/alphagov/govuk-prototype-kit/pull/1050)
+- [Pull request #1050: Do not swallow errors from session-data-defaults.js](https://github.com/alphagov/govuk-prototype-kit/pull/1050)
```

Comment:
Should we use code format to display `session-data-defaults.js`?

Reply:
Thanks @EoinShaughnessy , good shout. That seems to match up with that we've done elsewhere in the changelog [(example)](https://github.com/alphagov/govuk-prototype-kit/blame/main/CHANGELOG.md#L27)

Reply:
```suggestion
- [Pull request #1050: Do not swallow errors from `session-data-defaults.js`](https://github.com/alphagov/govuk-prototype-kit/pull/1050)
```